examples/drivers/windows/otLwf/radio.c

```diff
@@ -814,6 +814,14 @@ void otPlatRadioSetDefaultTxPower(_In_ otInstance *otCtx, int8_t aPower)
     }
 }
 
+int8_t otPlatRadioGetReceiveSensitivity(_In_ otInstance *otCtx)
+{
+    NT_ASSERT(otCtx);
+    PMS_FILTER pFilter = otCtxToFilter(otCtx);
+    UNREFERENCED_PARAMETER(pFilter);
+    return -100;
```

Comment:
I referred to [otPlatRadioGetRssi()](https://github.com/openthread/openthread/blob/master/examples/drivers/windows/otLwf/radio.c#L492), since they have similar functionality. And I also noticed that there are many other platform APIs have similar implementation.

So would you mind keep it as is, and update all these APIs together?